Resources/Locale/en-US/corvax/station-goal/station-goal-component.ftl

```diff
@@ -0,0 +1,324 @@
+    - Valuable, but not unique boards.
+
+    Monitor the safety of the contents in the storage until the end of the shift, a cleanup crew will come retrieve the contents as they prepare the station.
+    ════════════════════════════════════════
+
+station-goal-zoo=
+    ███╗░░██╗████████╗
+    ████╗░██║╚══██╔══╝ Form NT-No.{$station}-CC
+    ██╔██╗██║░░░██║░░░ Target Order
+    ██║╚████║░░░██║░░░ Date: {$date}
+    ██║░╚███║░░░██║░░░
+    ╚═╝░░╚══╝░░░╚═╝░░░
+    ════════════════════════════════════════
+    Dear station Command, the purpose of your shift is to improve the recreational value of the personnel at the station.
+
+    It is necessary to build a zoo with at least 5 enclosures containing different types of animals ordered from the supply department.
+    Provide animals with food, at least one cleaning robot in each enclosure, and everything necessary for life, depending on the type of animal.
+    It is also necessary to build a bathhouse for the animals, water vapor must be supplied by Atmospheric Technicians.
+
+    Upon completion of the zoo, it is required to provide the crew with at least 20 minutes of free time from work so that they can visit the new zoo.
+    ════════════════════════════════════════
+
+station-goal-labor=
+    ███╗░░██╗████████╗
+    ████╗░██║╚══██╔══╝ Form NT-No.{$station}-CC
+    ██╔██╗██║░░░██║░░░ Target Order
+    ██║╚████║░░░██║░░░ Date: {$date}
+    ██║░╚███║░░░██║░░░
+    ╚═╝░░╚══╝░░░╚═╝░░░
+    ════════════════════════════════════════
+    Dear station Command, the purpose of your shift is to increase the motivation of the personnel for the growth of labor productivity.
+
+    This requires that each of the heads during the shift closely monitors the performance of the duties of their employees and evaluates them.
+    After the time set by Command for evaluation, in each of the departments, the best, in the opinion of the head, employee should be selected, who will be invited to a dinner, where the Command staff will be obliged to award them a medal and a prize.
+    The heads must provide a report indicating the employee's position and merits for the shift.
+    Drinks and meals should be prepared for the dinner, as well as, if possible, several entertainment events that allow the presence of actors and musicians.
+    For the duration of the celebration, the dining room or other place chosen for the event must be inaccessible to the rest of the crew.
+
+    The duration of the shift for a more accurate assessment of the work of the personnel should be set by the Command staff.
+    After the dinner someone must announce the end of the shift and call the evacuation shuttle.
+    ════════════════════════════════════════
+
+station-goal-lectures=
+    ███╗░░██╗████████╗
+    ████╗░██║╚══██╔══╝ Form NT-No.{$station}-CC
+    ██╔██╗██║░░░██║░░░ Target Order
+    ██║╚████║░░░██║░░░ Date: {$date}
+    ██║░╚███║░░░██║░░░
+    ╚═╝░░╚══╝░░░╚═╝░░░
+    ════════════════════════════════════════
```

Comment:
This preface should be a single loc string, rather than being repeated each time.